jdbc/src/main/java/org/apache/zeppelin/jdbc/JDBCInterpreter.java

```diff
@@ -513,7 +513,7 @@ public Connection getConnection(String dbPrefix, InterpreterContext context)
     String url = properties.getProperty(URL_KEY);
     String connectionUrl = appendProxyUserToURL(url, user, dbPrefix);
 
-    String authType = properties.getProperty("zeppelin.jdbc.auth.type", "SIMPLE")
+    String authType = StringUtils.defaultIfEmpty(getProperty("zeppelin.jdbc.auth.type"), "SIMPLE")
```

Comment:
Is `StringUtils.defaultIfEmpty` necessary ? getProperty would get default value if it is not set.

Reply:
Without `StringUtils.defaultIfEmpty`, jdbcIntegrationTest will fail, see also:
https://github.com/apache/zeppelin/runs/1728251503

If `zeppelin.jdbc.auth.type` is empty string (`""`), `properties.getProperty("zeppelin.jdbc.auth.type", "SIMPLE")` will return `""`.
But `defaultIfEmpty(T str, T defaultStr)` returns the value of defaultStr the CharSequence is empty or null.

Reply:
@zjffdu DONE